packages/core/src/awsService/ec2/commands.ts

```diff
@@ -71,3 +72,10 @@ async function getSelection(node?: Ec2Node, filter?: instanceFilter): Promise<Ec
 export async function copyInstanceId(instanceId: string): Promise<void> {
     await copyToClipboard(instanceId, 'Id')
 }
+
+export async function openLogDocument(node?: Ec2InstanceNode): Promise<void> {
+    const uri = ec2LogSchema.form(await getSelection(node))
+    const doc = await vscode.workspace.openTextDocument(uri)
+    await vscode.window.showTextDocument(doc, { preview: false })
+    await vscode.languages.setTextDocumentLanguage(doc, 'log')
```

Comment:
I wonder if these 3 lines should be a util function in `shared/utilities/textDocumentUtilities.ts`. Then the call would look like:

 ```ts
textDocUtil.show(ec2LogSchema.form(await getSelection(node)))
 ```

Reply:
good catch.